tests/test_dsp.py

```diff
@@ -103,6 +103,24 @@ def test_group_delay_custom_frequencies(impulse_group_delay):
     npt.assert_allclose(grp, impulse_group_delay[1][0, frequency_idx])
 
 
+def test_linear_phase():
+    # test signal
+    N = 64
+    x = pf.signals.impulse(N)
+
+    # test default parameters
+    y = dsp.linear_phase(x, N/2)
+    # test output
+    assert isinstance(y, pf.Signal)
+    npt.assert_allclose(dsp.group_delay(y), N / 2 * np.ones(y.n_bins))
+    # test if input did not change
+    npt.assert_allclose(x.time, pf.signals.impulse(N).time)
+
+    # test assertion
+    with pytest.raises(TypeError, match="signal must be a pyfar Signal"):
+        dsp.linear_phase(1, 0)
+
```

Comment:
Missing test for multichannel signal

Reply:
I added testing and the possibility to pass array likes for `group_delay`